### DISCUSSION TITLE
Radeon ProRender for Blender 3.3

### DISCUSSION THREAD

**Discussion by bsavery** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#379] [Announcements]
Radeon ProRender for Blender 3.2.2

This release is mainly adds support for smoke, fire, and fog volume objects in RPR 2.0 mode.  Additionally, the Volume Scatter, Volume Info and Principled Volume shader are supported.

For more information on changes and fixes see the [Release Notes](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/blob/master/Changelog.md#version-33)

Supports Blender versions 2.8X and 2.90, 2.91, 2.92, and 2.93

<hr /><em>This discussion was created from the release <a href='https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/releases/tag/v3.3'>Radeon ProRender for Blender 3.3</a>.</em>

**Comment by blenderdiy** [comment]
prorender has no interest in changing rays4.1, change rocm/hip, amd graphics card strong future not bad user nv side there are too many renderers to choose from, amd side than for the rival gpu sacrifice their gpu speed experience

**Reply by bsavery** [reply]
?  I'm not sure I quite follow this comment.  Is this a question, request or just a complaint?

**Comment by KK64tam** [comment]
Hi, I can't use ProRender3.3 and show the error . Can anyone tell me why and how to fix it
![IMG_2413](https://user-images.githubusercontent.com/95188258/143783848-bb1bf4e6-6209-474a-8dfc-87f2a7fbf339.jpg)
 ?

**Reply by cykhu** [reply]
did you solve the problem?

**Comment by KK64tam** [comment]
Hi, same problems still have:
Don't show display in system
[image: Screenshot 2021-11-30 at 3.05.46 PM.png]
error when click Radeon ProRender after installed
[image: Screenshot 2021-12-14 at 11.10.32 PM.png]

cykhu ***@***.***> 於 2021年12月14日 週二 上午11:57寫道：
